bookkeeper-server/src/test/java/org/apache/bookkeeper/server/TestEmbeddedServer.java

```diff
@@ -1,60 +1,67 @@
 /*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
+ * Copyright 2022 The Apache Software Foundation.
```

Comment:
The license seems wrong

Reply:
Changed with a copy took from other files